cylc/flow/ws_data_mgr.py

```diff
@@ -361,22 +419,41 @@ def update_task_proxies(self, task_ids=None):
                 prereq_obj = prereq.api_dump(self.workflow_id)
                 if prereq_obj:
                     prereq_list.append(prereq_obj)
+            del tproxy.prerequisites[:]
             tproxy.prerequisites.extend(prereq_list)
-            for _, msg, is_completed in itask.state.outputs.get_all():
-                tproxy.outputs.append(f"{msg}={is_completed}")
+            tproxy.outputs[:] = [
+                f'{msg}={is_completed}'
+                for _, msg, is_completed in itask.state.outputs.get_all()
```

Comment:
Sorry, I didn't notice this last time, but I guess we are happy to use the message of the output and not the trigger?

Reply:
Well I think the trigger is an internal mechanism;
```
Arguments:
   [REG]                                  Suite name
   [TASK-JOB]                             Task job identifier CYCLE/TASK_NAME/SUBMIT_NUM
   [[SEVERITY:]MESSAGE ...]               Messages
```
we can use a protobuf repeated message or mapping to add more information..

Reply:
The trigger is part of the graph... In the following example, `eggs` is the *trigger*, `laid many eggs` is the `message`. For graphing purpose, you probably want the trigger instead of the message.

```
[scheduling]
    [[graph]]
        R1 = chicken:eggs => market
[runtime]
    [[chicken]]
        [[[outputs]]]
            eggs = laid many eggs
```